src/http.rs

```diff
@@ -14,19 +14,30 @@ use crate::{
     error::ApiError,
 };
 
+pub const API_PORT_ENV: &str = "MEMBRANE_VIDEO_COMPOSITOR_API_PORT";
+const PORT_AVAILABLE_MESSAGE: &str = "Make sure that used port is available. For specifying port use: MEMBRANE_VIDEO_COMPOSITOR_API_PORT environment variable.";
```

Comment:
I'm not really sold on that error message. If the issue is different then it will only confuse users.